autoscaling/autoscaling.go

```diff
@@ -298,6 +355,7 @@ func (autoscaling *AutoScaling) CreateLaunchConfiguration(options *CreateLaunchC
 		b64.Encode(userData, []byte(options.UserData))
 		params["UserData"] = string(userData)
 	}
+	addBlockDeviceParams("", params, options.BlockDevices)
```

Comment:
If you're always passing an empty string for the first arg, should we just drop the `prename` arg?